src/Misc/SegmentedProgressBar.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { ReactElement, useCallback } from 'react';
+import styled, { keyframes } from 'styled-components';
+
+const TooltipText = styled.span<{width?: string}>`
+  visibility: hidden;
+  padding: 5px;
+  font-size: 10px;
+  color: #000000;
```

Comment:
I think it would be nice a text color similar to what we are using, for this tooltip I think the closer will be the font color of the dropdown options  hsla(0, 0%, 56.1%, 1.000);

color: hsla(0, 0%, 56.1%, 1.000);